### ISSUE TITLE
How to let jemalloc overrides system allocator

### ISSUE THREAD

**Issue by BusyJay** [alexcrichton/jemallocator#39]
This crate builds jemalloc with an explicit prefix, which prevents it from overriding system allocator widely used behind FFI. So for a program that uses a lot of C/C++ APIs, it's hard to make it get benefits from jemalloc.

Also note that the build-in rust jemalloc allocator only uses prefix when it has to.

Any ideas?

**Comment by alexcrichton** [comment]
Definitely! I'd be totally down for a cargo feature that unprefixes the symbols.
